radar/web/src/backend/connection.ts

```diff
@@ -151,6 +151,7 @@ export type RadarUpdate = {
 export type RadarState = {
     players: RadarPlayerInfo[],
     worldName: string,
+    bomb: RadarBombInfo,
```

Comment:
Maybe use an Option<RadarBombInfo> as bomb info can be null if there is no bomb available. This also replaces the need for the magic `-6969` "empty" value.